frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/common/FastForwardMerge.java

```diff
@@ -38,7 +38,7 @@ private static void mergeNonCurrentBranch(Project project,
     val successFFMergeNotification = fetchNotificationPrefix
         + getString("action.GitMachete.BaseFastForwardMergeBranchToParentAction.notification.title.ff-success")
             .format(stayingName, movingName);
-    val failFFMergeNotification = fetchNotificationPrefix
+    val failFFMergeNotification = fetchNotificationPrefix.replaceAll("<br/>", getString("utils.new-line-character"))
```

Comment:
Any specific reason for extracting `\n` to properties? 🤔 esp. given that `<br/>` isn't

Reply:
<img width="946" alt="Screenshot 2022-06-06 at 14 29 47" src="https://user-images.githubusercontent.com/33667003/172160958-cd8b9e74-a8e8-4335-9bc7-79855d1c3f03.png">

😞

Reply:
On UNIX systems it should return \n, but on Windows it might return a different value

Reply:
:ups: right indeed... just `@SuppressWarnings("regex")` then

Reply:
Or indeed, use `System.lineSeparator()` 🤔

Reply:
but on Windows System.lineSeparator can return “\r\n”  instead of \n